### DISCUSSION TITLE
Stable version

### DISCUSSION THREAD

**Discussion by Inscure** [tabler/tabler#1971] [General]
Hello,

First of all, thank you for the great work on Tabler! We are very interested in using the tool in our projects. However, we have noticed that only beta versions are available at the moment.

Our company's policy requires us to use stable releases for production environments. Could you kindly let us know when you plan to release a stable version of Tabler?

We would love to adopt it as soon as it's available.

Thank you in advance for your response!

**Comment by Inscure** [comment]
I also noticed that beta15 is marked as the latest release on the [Tabler GitHub page](https://github.com/tabler/tabler). However, I believe the most recent version is actually beta21. 

<img width="270" alt="image" src="https://github.com/user-attachments/assets/de211f29-51c4-4c95-8fff-b8a024c8429c">

<img width="983" alt="image" src="https://github.com/user-attachments/assets/75b50c51-294d-4da5-9c12-a5c1cc42c6b2">

https://github.com/tabler/tabler/releases

**Comment by gjanezic** [comment]
Also, there's no dist/ directory in -beta21. Could you make a release including dist/?

**Reply by William-Cano** [reply]
Hi, I had the same problem of not finding the dist folder, I solved it by downloading from the official page https://tabler.io/admin-template it will take you to a payment option at the bottom of the page it says free download

**Reply by gjanezic** [reply]
Thanks for the replay, but looks to me that's -beta20..

**Comment by dylan-tao** [comment]
> Hello,
> 
> First of all, thank you for the great work on Tabler! We are very interested in using the tool in our projects. However, we have noticed that only beta versions are available at the moment.
> 
> Our company's policy requires us to use stable releases for production environments. Could you kindly let us know when you plan to release a stable version of Tabler?
> 
> We would love to adopt it as soon as it's available.
> 
> Thank you in advance for your response!

@Inscure @gjanezic @William-Cano @beeman   I have sent the author an email with the updated version, but I haven't replied yet. Please reply after seeing it. I checked whether the submission record was specially deleted in the dist or 21 version. I don't know the author's intention.

@codecalm Please answer this question, thanks！Does the finished product package of the official website need to be updated to the latest version of 21?

**Reply by OgreTransporter** [reply]
It would be nice if the package from NPM (https://www.npmjs.com/package/@tabler/core) could always be attached to the corresponding release on GitHub, e.g. as `dist.zip`.
